### ISSUE TITLE
Multi Forms break the layout

### ISSUE THREAD

**Issue by tomsiodlak** [simplebits/Pears#24]
I copied both the HTML and CSS for both of the multi forms from the official Pears site and this is what the post ends up looking like: http://cl.ly/3f1N0Y0k3u3H0C3L2L2i

**Comment by shystie** [comment]
I'm having the same problem. It seems that the HTML is being rendered after this line:
<fieldset>
        <label for="bio">Bio</label>
        <textarea id="bio">

**Comment by jeremyzilar** [comment]
Yeah, Dan said in his post that he was having trouble with <textarea> inside a <textarea>.
